renovate.json

```diff
@@ -0,0 +1,45 @@
+{
+  "extends": ["config:base", "group:allNonMajor", "schedule:earlyMondays"],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["*"],
+      "rangeStrategy": "update-lockfile"
+    },
+    {
+      "matchPackageNames": [
```

Comment:
question: long term, do we want to reduce the number of packages we exclude from renovate?